bin/saya/src/args/starknet_account.rs

```diff
@@ -0,0 +1,28 @@
+//! Data availability options.
+
+use clap::Args;
+use katana_primitives::FieldElement;
+use url::Url;
+
+#[derive(Debug, Args, Clone)]
+pub struct StarknetAccountOptions {
+    #[arg(long)]
+    #[arg(env)]
+    #[arg(help = "The url of the starknet node.")]
+    pub starknet_url: Url,
+
+    #[arg(long)]
+    #[arg(env)]
+    #[arg(help = "The chain id of the starknet node.")]
+    pub chain_id: String,
```

Comment:
I'm wondering if we need the chain id. As the network is forked (to access the state of the settlement layout), isn't the chain id something we should already have from the Katana we're pointing to?

Hum, we should keep it, after thinking about this, if we are running Katana as a L3 with core contract, the chain-id will differ. So let's keep it for now. :)